### ISSUE TITLE
pip 18.1 issues

### ISSUE THREAD

**Issue by auludag** [rehmatworks/serverpilot-letsencrypt#26]
hi,

when i see "You are using pip version 8.1.1, however version 18.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command." i did that.

but after that pip command regarding rwssl seems broken

Traceback (most recent call last):
  File "/usr/bin/pip", line 9, in <module>
    from pip import main
ImportError: cannot import name main

**Comment by onetrev** [comment]
Similar errors for me when I run `rwssl` on a server running pip-18.1

```
Traceback (most recent call last):
  File "/usr/local/bin/rwssl", line 7, in <module>
    from rwssl.rwssl import main
  File "/usr/local/lib/python2.7/dist-packages/rwssl/rwssl.py", line 7, in <module>
    import validators
  File "/usr/local/lib/python2.7/dist-packages/validators/__init__.py", line 1, in <module>
    from .between import between  # noqa
  File "/usr/local/lib/python2.7/dist-packages/validators/between.py", line 2, in <module>
    from .utils import validator
  File "/usr/local/lib/python2.7/dist-packages/validators/utils.py", line 5, in <module>
    import six
ImportError: No module named six
```

**Comment by onetrev** [comment]
But I did get it working again like so...
- I downgraded to Pip 9.0.1 as I knew that worked on another server. I did that with this command: `sudo pip install --force-reinstall pip==9.0.``
- What's odd though is it still didn't work. So I then uninstalled `rwssl` per documentation.
- Then I re-installed `rwssl` and now it works. Keep in mind I'm still running pip 9.0.1

Hope that at least helps get you back and running with this very useful script!
